components/Button/index.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+export const Button = ({ setProduct, produkt, productToBuy }) => {
```

Comment:
adjust prop according to changes made while passing it down

and fix typo in "produkt"

```suggestion
export const Button = ({ setProductToBuy, product, productToBuy }) => {
```